src/Core/AuthenticationHelpers/AuthenticationSimulator/SimulatorAuthenticationHandler.cs

```diff
@@ -85,7 +85,7 @@ protected override Task<AuthenticateResult> HandleAuthenticateAsync()
 
         // AuthenticationTicket is Asp.Net Core Abstraction of Authentication information
         // of the authenticated user.
-        AuthenticationTicket ticket = new(claimsPrincipal, EasyAuthAuthenticationDefaults.AUTHENTICATIONSCHEME);
+        AuthenticationTicket ticket = new(claimsPrincipal, SimulatorAuthenticationDefaults.AUTHENTICATIONSCHEME);
```

Comment:
why this change?

Reply:
It didn't make sense that `EasyAuthAuthenticationDefaults.AUTHENTICATIONSCHEME` was used to denote the scheme used to authenticate a Simulator request. 

Looking at source: https://github.com/dotnet/aspnetcore/blob/3f1acb59718cadf111a0a796681e3d3509bb3381/src/Http/Authentication.Abstractions/src/AuthenticationTicket.cs

I don't observe the authenticationScheme serving functional purpose besides acting as metadata.